src/app/auth/login/login.component.ts

```diff
@@ -11,32 +10,30 @@ import { AuthService } from '../auth.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LoginComponent implements OnInit {
-  loginForm: FormGroup;
-  showPassword = false;
+  public loginForm: FormGroup;
+  public showPassword: boolean = false;
```

Comment:
Equal `false` by default. So it is clear that this boolean, no?

Reply:
Equals `undefined` by default. Yeah, it's clear but let's write all class properties in one style (with type definitions) I believe it'll look neater

Reply:
And I think it's better to use `isShowPassword `name for this kind of variables

Reply:
Maybe yes